components/zcash_address/src/kind/unified.rs

```diff
@@ -299,19 +252,17 @@ impl TryFrom<Vec<Receiver>> for Address {
             Err(ParseError::OnlyTransparent)
         } else {
             // All checks pass!
-            Ok(Address(receivers))
+            Ok(Self::from_inner(receivers))
         }
     }
-}
 
-impl Address {
-    /// Returns the raw encoding of this Unified Address.
-    pub(crate) fn to_bytes(&self, hrp: &str) -> Vec<u8> {
+    /// Returns the raw encoding of this Unified Address or viewing key.
+    fn to_bytes(&self, hrp: &str) -> Vec<u8> {
```

Comment:
Hmm, this was previously an internal method, and now it is exposed publicly. I'd prefer that it be placed on `Sealed`.